content/pt_PT/es6.md

```diff
@@ -1,33 +1,35 @@
-# ES6 on io.js
+# ES6 no io.js
 
-io.js is built against modern versions of [V8](https://code.google.com/p/v8/). By keeping up-to-date with the latest releases of this engine we ensure new features from the [JavaScript ECMA-262 specification](http://www.ecma-international.org/publications/standards/Ecma-262.htm) are brought to io.js developers in a timely manner, as well as continued performance and stability improvements.
+O io.js é construído com versões modernas do [V8](https://code.google.com/p/v8/). Mantendo-nos atualizados com a última versão deste motor, garantimos que as novas funcionalidades da [especificação JavaScript ECMA-262](http://www.ecma-international.org/publications/standards/Ecma-262.htm) são disponibilizadas rapidamente aos programadores io.js, bem como as melhorias de performance e estabilidade.
```

Comment:
I agree. Fixed.

(we should discuss in english here so that PR reviewers understand what's going on :) )